mobilecoind/src/payments.rs

```diff
@@ -763,8 +765,13 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
             fog_resolver_factory(&fog_uris).map_err(Error::Fog)?
         };
 
+        // FIXME: We should get the block version from
+        // `LedgerDb::get_last_block()?.version`
+        let block_version = BlockVersion::ONE;
```

Comment:
I would not leave this as a TODO. While libmobilecoin and the android bindings are used by the SDKs and changing them requires coordinating with them, this is a standalone change in code that entirely lives in this repo and nothing externally would be affected by it getting changed.

Reply:
ok i'll figure out how to do it

Reply:
it turned out to be pretty easy, just a few lines :)